initrd/bin/cbfs-init

```diff
@@ -9,16 +9,22 @@ if [ -z "$CONFIG_PCR" ]; then
 	CONFIG_PCR=7
 fi
 
+if [ "$CONFIG_CBFS_VIA_FLASHROM" = "y" ]; then
+	# Use flashrom directly, because we don't have /tmp/config with params for flash.sh yet
+	/bin/flashrom -p internal --fmap -i COREBOOT -i FMAP -r /tmp/cbfs-init.rom > /dev/null 2>&1
```

Comment:
I hope this never fails... Some error check would be great. We blindly assume `/tmp/cbfs-init.rom` will be created (meaning successfully read). I.e. add some `|| die ...`  in the line with flashrom command

Reply:
Added error check in https://github.com/Dasharo/heads/pull/7/commits/2429359dfe37ffacb6390cf3547ee1483168e9a3 . If the command fails, CBFS_ARG won't be set, so previous behavior will be kept (trying to read from the 16MiB region)